ale/drivers/cassini_drivers.py

```diff
@@ -96,7 +95,7 @@ def focal2pixel_samples(self):
         """
         # Microns to mm
         pixel_size = spice.gdpool('INS{}_PIXEL_SIZE'.format(self.ikid), 0, 1)[0] * 0.001
-        return [0.0, 1/pixel_size, 0.0]
+        return [0.0, -1/pixel_size, 0.0]
```

Comment:
Correct me if I'm wrong, these control the transform to/from detector position and focal plane x,y. Changing the sign on these coefficients cause that element of the transform to be subtracted instead of added. This is opposite to what the ISIS focal plane map is doing. Can this be explained?

Reply:
We already covered this in person, but just to document things here: we needed to do an extra 180 rotation to the final sensor orientation. The negatives here were basically faking that rotation.